.gitignore

```diff
@@ -1,2 +1,6 @@
 .ox3rc
 *.pyc
+ox3apiclient.egg-info/
+landing/*
+build/*
+dist/*
```

Comment:
these folders look specific to your environment. Maybe you want to change your ~/.gitignore instead.